### PR TITLE
Make 'as' prop in Controller examples consistent

### PR DESCRIPTION
**Pull request by orteth01**
in `RHFInput.ts`, it [specifies a preffered syntax fo the Controller component](https://github.com/react-hook-form/react-hook-form-website/blob/master/src/components/codeExamples/RHFInput.ts#L17-L18). This PR updates all controller examples to match the preferred syntax.

it also updates all instances of `RHFInput` to `Controller`.

### CODE REVIEW

File: src/components/codeExamples/RHFInput.ts [hunk 1]
```diff
@@ -30,6 +30,7 @@ function App() {
         defaultValue={{ value: 'chocolate' }}
       />
       
+      {* // what is this example showing? can we remove it? *}
```
Comment (orteth01) [inline src/components/codeExamples/RHFInput.ts]: don't merge w/ this here. i just wanted to flag the `<Controller />` below. i'm not really sure what it is there for.

**Comment by bluebill1049** [comment]
awesome stuff, i will take a look during lunchtime ❤️ much appreciated.

File: src/components/codeExamples/RHFInput.ts (hunk 1 above)
Comment (bluebill1049) [reply src/components/codeExamples/RHFInput.ts]: This file is showing under Controller section, if u search the file usage. I should rename the file to be Controller

Comment (orteth01) [reply src/components/codeExamples/RHFInput.ts]: I can rename the file 👍

Can i remove this last `Controller` in the example (lines 34-39)? They don't show an example of anything new, i think

File: src/components/codeExamples/reactNativeController.ts [hunk 2]
```diff
@@ -1,27 +1,34 @@
-export default `import React from 'react';
-import { View, TextInput, Button } from 'react-native';
-import { useForm, Controller } from 'react-hook-form';
+export default `import React from "react";
```
Comment (orteth01) [inline src/components/codeExamples/reactNativeController.ts]: both `reactNativeController` and `reactNativeRHFInput` existed, but they were essentially showing the same thing. i decided to use `reactNativeRHFInput`.

so the old `reactNativeController` is *deleted*
and then i renamed `reactNativeRHFInput` to `reactNativeController`.

sorry if this is unnecessarily confusing

Comment (bluebill1049) [reply src/components/codeExamples/reactNativeController.ts]: Oh sweet awesome.

File: src/components/codeExamples/RHFInput.ts (hunk 1 above)
Comment (bluebill1049) [reply src/components/codeExamples/RHFInput.ts]: Yea sure

**Pull request merged**